### DISCUSSION TITLE
Getting errors while building

### DISCUSSION THREAD

**Discussion by vishalshrm539** [webpack/webpack#15856] [General]
Hi,

I am getting this error when I try to use private fields in a .js file within an Angular Project.

Error: ./src/app/_helpers/sample.js 6:11
Module parse failed: Unexpected character '#' (6:11)
File was processed with these loaders:
  ./node_modules/@angular-devkit/build-angular/src/babel/webpack-loader.js
  ./node_modules/@ngtools/webpack/src/ivy/index.js
  ./node_modules/source-map-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.

sample.js:
class Sample {

constructor() {
this.#reloadConfig(); // 6th line
}

#reloadConfig() {
}
...

**Comment by alexander-akait** [comment]
Plese use an issue template, what is webpack version?

**Reply by alexander-akait** [reply]
Also look like you miss `}` for your class, to close it

**Reply by vishalshrm539** [reply]
Webpack version is: 4.46.0 .

Regarding that missing semi-colon, I probably would have missed it while copying, it is fine in my code.

**Reply by vishalshrm539** [reply]
FYI, I'm using Angular 11 if at all that matters.

**Reply by alexander-akait** [reply]
webpack v4 doesn't support private fields, you need webpack v5 to handle them

**Reply by vishalshrm539** [reply]
Was able to resolve this issue by upgrading Angular from 11 to 12, and I still have Webpack V4 only.

**Reply by vishalshrm539** [reply]
FYI, Upgrading Webpack to v5 alone didn’t help either.
